recipes/cloudwatch_agent_config.rb

```diff
@@ -14,6 +14,10 @@
 # or in the "LICENSE.txt" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
 # OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions and
 # limitations under the License.
+
+# Install cloudwatch agent if not already installed
+include_recipe 'aws-parallelcluster::cloudwatch_agent_install'
```

Comment:
what I do not like here is that the cloudwatch_agent_install is re-executed at runtime when using a PCluster AMI, and some actions inside this recipe doesn't have a guard and are re-executed